config/initializers/rack_attack.rb

```diff
@@ -24,6 +24,10 @@ class Rack::Attack
     req.ip if req.path == '/v0/vic/vic_submissions' && req.post?
   end
 
+  throttle('evss_claims_async', limit: 10, period: 1.minute) do |req|
```

Comment:
So, best case scenario only 10 users can hit this route per minute (which is polled 12 times per minute per user by the frontend)?
Seems low to me.

Reply:
I thought this meant that one ip address can hit the endpoint only 10 times per minute.

Reply:
Ah, you're right.  That's what the line below does.  Perhaps we should raise it to `12` to match the frontend polling.